src/main/java/de/tracetronic/jenkins/plugins/ecutest/log/TTConsoleLogger.java

```diff
@@ -87,7 +87,7 @@ public void logComException(final Exception exception) {
         logError(String
             .format("Caught ComException: %s%n"
                     + "For further information see FAQ: "
-                    + "https://wiki.jenkins-ci.org/x/joLtB#TraceTronicECU-TESTPlugin-FAQ",
+                    + "https://github.com/jenkinsci/ecutest-plugin#faq",
```

Comment:
May be add also link to debugging section? https://github.com/jenkinsci/ecutest-plugin#debugging